src/KubernetesClient/KubernetesClient.csproj

```diff
@@ -9,7 +9,8 @@
     <PackageIconUrl>https://raw.githubusercontent.com/kubernetes/kubernetes/master/logo/logo.png</PackageIconUrl>
     <PackageTags>kubernetes;docker;containers;</PackageTags>
 
-    <TargetFrameworks>netstandard1.4;net452;netcoreapp2.1;xamarinios10;monoandroid81</TargetFrameworks>
+    <TargetFrameworks>netstandard1.4;net452;netcoreapp2.1</TargetFrameworks>
+    <!-- <TargetFrameworks>netstandard1.4;net452;netcoreapp2.1;xamarinios10;monoandroid81</TargetFrameworks> -->
```

Comment:
There are still people interested in this for xamarin, can we retain it for xamarin?

Reply:
Azure pipeline windows image does not have ios sdk on it. Build will fail only on CI.
My idea is to create another issue for it and ping Azure pipeline team.